### PR TITLE
flashrom: Add support for Apple Silicon from https://github.com/flash…

### PR DESCRIPTION
**Pull request by areese**
…rom/flashrom/pull/212/commits/0c7b279d78f95083b686f6b1d4ce0f7b91bf0fd0

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/flashrom.rb
Comment (carlocab) [inline Formula/flashrom.rb:26]: Can we add a link to your [PR](https://github.com/flashrom/flashrom/pull/212) here? We probably also want to wait for feedback from upstream before merging this.

Comment (areese) [reply Formula/flashrom.rb:26]: Yes I missed linking the or it needs to be a comment to merge

Comment (areese) [reply Formula/flashrom.rb:27]: Done.

**Review by Moisan — APPROVED** [review]
Thank you @areese !

**Comment by areese** [comment]
Yep
I need to check if it got upstreamed
I haven’t gone back through the ones I added to clean them up I’ll try soon I hope

**Pull request merged**
